spec/search_history_spec.rb

```diff
@@ -1,15 +1,49 @@
 require 'spec_helper'
 
 RSpec.describe Kitchen::SearchHistory do
+  context 'init' do
+    it 'creates instance variables' do
+      sh = described_class.new('foo', 'bar')
+      expect(sh.upstream).to eq('foo')
+      expect(sh.latest).to eq('bar')
+    end
+  end
+
   context 'empty' do
     it 'converts to an empty array' do
       expect(described_class.empty.to_a).to eq []
     end
   end
 
+  context '#add' do
+    it 'adds nil if given nil value' do
+      expect(described_class.empty.add(nil).to_a).to eq [nil]
+    end
+  end
+
   context '#to_s' do
     it 'works' do
       expect(described_class.empty.add(nil).add("foo").add([".blah",".bar"]).to_s).to eq "[?] [foo] [.blah, .bar]"
     end
   end
+
+  context '#to_a' do
+    it 'returns [latest] if no upstream' do
+      expect(described_class.empty.add('foo').to_a).to eq(['foo'])
+    end
+
+    it 'returns array with upstream and latest' do
+      expect(described_class.empty.add('foo').add(["blah","bar"]).add(['baz']).to_a).to eq(['foo','blah, bar','baz'])
```

Comment:
rubocop should be complaining about the double quoted strings, changing them to single quoted.